feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/domain/interactor/GovernancePushSettingsInteractor.kt

```diff
@@ -28,16 +29,17 @@ class RealGovernancePushSettingsInteractor(
     override fun governanceChainsFlow(): Flow<List<ChainWithGovTracks>> {
         return chainRegistry.currentChains
             .map {
-                it.filter { it.supportProxy }
-                    .flatMap { it.chainToGovTypes() }
+                it.filter { it.pushSupport }
+                    .flatMap { it.supportedGovTypes() }
                     .map { (chain, govType) -> ChainWithGovTracks(chain, govType, getTrackIds(chain, govType)) }
                     .sortedWith(Chain.defaultComparatorFrom(ChainWithGovTracks::chain))
             }
     }
 
-    private fun Chain.chainToGovTypes(): List<Pair<Chain, Chain.Governance>> {
-        return governance.filter { it == Chain.Governance.V2 }
-            .map { this to it }
+    private fun Chain.supportedGovTypes(): List<Pair<Chain, Chain.Governance>> {
+        return openGovIfSupported()?.let {
+            listOf(this to it)
+        }.orEmpty()
```

Comment:
I guess `listOfNonNull(openGovIfSupported()?.let { this to it })` is a lttile bit cleaner